### ISSUE TITLE
Freeze with empty list of pages

### ISSUE THREAD

**Issue by bigluck** [maxep/MXSegmentedPager#57]
Hi @maxep, you can reproduce this bug by adding this code in the `MXSegmentedPager-Swift` sample:

``` Swift
override func numberOfPagesInSegmentedPager(segmentedPager: MXSegmentedPager) -> Int {
    return 0
}
```

Thanks,
Luca

**Comment by maxep** [comment]
Hi @bigluck 

There is indeed an infinite loop in MXPagerView that can easily be fixed. But anyway, HMSegmentedControl do not support an empty array of titles, it crashes... So I'm thinking about adding an assertion in that case.

**Comment by bigluck** [comment]
:+1:

**Comment by maxep** [comment]
Fixed in 3.0.4
